fe/fe-core/src/main/java/org/apache/doris/catalog/ResourceMgr.java

```diff
@@ -109,8 +112,41 @@ public int getResourceNum() {
         return nameToResource.size();
     }
 
-    public List<List<String>> getResourcesInfo() {
-        return procNode.fetchResult().getRows();
+    public List<List<Comparable>> getResourcesInfo(String name, boolean accurateMatch, Set<String> typeSets) {
+        List<List<String>> targetRows = procNode.fetchResult().getRows();
+        List<List<Comparable>> returnRows = Lists.newArrayList();
+
+        for (List<String> row : targetRows) {
+            if (row == null || row.size() < 2) {
+                continue;
+            }
+
+            String resourceName = row.get(0);
+            String resourceType = row.get(1);
+
+            if (name != null) {
+                if (accurateMatch && !resourceName.equals(name)) {
```

Comment:
equalsIgnore()

Reply:
There is need need ignore. The Name should be case sensitive